04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Please rename this note to add `.md` to the filename

Reply:
oops - sorry for the duplicate comment

Reply:
Done :)